pkgs/top-level/all-packages.nix

```diff
@@ -34533,6 +34533,8 @@ with pkgs;
 
   osmo = callPackage ../applications/office/osmo { };
 
+  osmo-rtl-sdr = callPackage ../by-name/os/osmo-rtl-sdr/package.nix { };
```

Comment:
You still need to remove this line. The point of `pkgs/by-name` is to de-clutter `all-packages.nix`. 
See https://github.com/NixOS/rfcs/blob/master/rfcs/0140-simple-package-paths.md